workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -11,5 +11,29 @@ public class SpatialOSComponent : MonoBehaviour
         public Entity Entity;
         public World World;
         public WorkerSystem Worker;
+
+        private EntityManager entityManager;
+
+        public bool TryGetGameObjectForEntity(EntityId entityId, out GameObject linkedGameObject)
+        {
+            linkedGameObject = default(GameObject);
+            if (!Worker.TryGetEntity(entityId, out var entity))
+            {
+                Worker.LogDispatcher.HandleLog(LogType.Warning, new LogEvent("Could not find ECS entity for given SpatialOS entity ID")
```

Comment:
Since this is a `Try` method, why are we logging here and below?

Reply:
simply to differentiate why the try failed, but if you prefer to not have logs in this method, we can remove them

Reply:
I would be really surprised to have a `Try` method log anything. Especially if I were to use it in speculative code that happens quite often, and accidentally end up spamming my deployment logs.

Reply:
will remove it :)